docs/src/theme/Navbar/index.js

```diff
@@ -226,7 +226,7 @@ function Navbar() {
                   [styles.hideLogoText]: isSearchBarExpanded,
                 })}>
                 {title}
-                <span className="badge badge--info">V2 alpha</span>
+                <span className="badge badge--info">V2 stable</span>
```

Comment:
I think we should remove this badge, it stays when you're browsing v1 docs. This confuses people.